package.json

```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "@openmined/sonar": "git+https://git@github.com/OpenMined/Sonar.git#f687278",
     "commander": "^2.11.0",
+    "geth": "gavinuhma/geth.js#ask-for-password",
```

Comment:
Any chance we can rely on the official version?

Reply:
I'm going to send them a pull request. They only support password files for unlocking, rather than the normal geth options